src/controllers/authentication.js

```diff
@@ -16,15 +18,42 @@ const createUser = async (req, res) => {
   };
   try {
     const user = await models.User.create(userData);
-    const {password, ...data} = user.toJSON();
-    const token = tokenGenerator({ userId: user.id });
+    const token = tokenGenerator({ userId: user.id }, { expiresIn: "1d" });
+    const url = `http://localhost:4000/user/verify_email/${token}`;
+    sendMail(
+      user.email,
+      "Email Verification",
+      "you can now verify your account",
+      url
+    );
     return res.status(201).json({
       message: "Account created successfully",
-      data: data,
-      token: token,
+      data: user,
+      token,
     });
   } catch (error) {
-    return res.status(500).json({ message: error });
+    console.log(error);
+    return res.status(500).send(error);
```

Comment:
Please remove this console.log, and never send plain text to the user, make sure that you are sending JSON objects